compiler-core/src/parse.rs

```diff
@@ -1270,9 +1271,18 @@ where
                 body,
                 return_type: (),
                 return_annotation,
+            })),
+            None =>  Ok(Some(Statement::Fn {
+                doc: None,
+                location: SrcSpan { start, end },
+                end_position: rbr_e - 1,
+                public,
+                name,
+                arguments: args,
+                body: UntypedExpr::Todo { location: SrcSpan {start, end}, label: None },
```

Comment:
I wasn't sure what to put for the location so I chose to use `start` and `end` here. Lmk if there are better options.